generated/provider_dependencies.json

```diff
@@ -231,7 +231,9 @@
     "deps": [
       "sqlparse>=0.4.2"
     ],
-    "cross-providers-deps": []
+    "cross-providers-deps": [
+      "openlineage"
```

Comment:
Does this make it a required dep or optional?

Reply:
I think we need to revert this PR.

This change add `openlineage` extra to `common.sql` but  `openlineage` has never been released. We can not add such extra till we release `openlineage`

Reply:
@eladkal So, I think we have chicken and egg problem - the OL provider is not released yet, but it's useless to release without any provider support. Can we not have those changes - which do not change what `common.sql` does, just add methods to it - without releasing OL provider first?

Reply:
We often have cases where providers co-depended. In such senario we release the providers in the same wave so users gain the functionality together.

So this PR should have been put on hold till OL provider is ready then merge them in a sequence so both are ready for the same release wave. In providers release cycle we release from main branch we do not cherry pick commits thus anything merged will be released.


Maybe @potiuk have another idea?